rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -72,226 +72,26 @@ namespace rocal{
     py::object wrapper_image_name_length(RocalContext context, py::array_t<int> array)
     {
         auto buf = array.request();
-        int* ptr = (int*) buf.ptr;
+        int* ptr = static_cast<int *>(buf.ptr);
         // call pure C++ function
-        int length =rocalGetImageNameLen(context,ptr);
+        int length = rocalGetImageNameLen(context,ptr);
         return py::cast(length);
     }
 
-    py::object wrapper_image_name(RocalContext context,  int array_len)
+    py::object wrapper_image_name(RocalContext context, int array_len)
     {
         py::array_t<char> array;
         auto buf = array.request();
-        char* ptr = (char*) buf.ptr;
-        ptr = (char *)calloc(array_len, sizeof(char));
+        char* ptr = static_cast<char *>(buf.ptr);
+        ptr = static_cast<char *>(calloc(array_len, sizeof(char)));
         // call pure C++ function
         rocalGetImageName(context,ptr);
```

Comment:
add space between arguments and follow same coding style throughout

Reply:
We are using the same coding style for all static_casts in C++ source files as well.